src/parser/jobs/sge/modules/Gauge.tsx

```diff
@@ -210,7 +210,7 @@ export class Gauge extends CoreGauge {
 	//#endregion
 
 	private onComplete() {
-		const addersgallLeniency = ADDERSGALL_TIME_REQUIRED / 2
```

Comment:
I'm not seeing the issue you mentioned in the PR notes in production at the moment, and I don't think this change is accurate. Can we drop it?

Reply:
okie removed!
(it's been a while since I looked at this, but I think it was a dev only issue, so if you happen to see it again, at least it won't come as a surprise lol)